glocal_exploration_ros/src/glocal_system.cpp

```diff
@@ -112,10 +112,20 @@ void GlocalSystem::mainLoop() {
   run_srv_ = nh_private_.advertiseService("toggle_running",
                                           &GlocalSystem::runSrvCallback, this);
 
+  // Limit the maximum update frequency.
+  // NOTE: This is mainly intended to avoid spinning at unlimited rates in case
+  //       a loopIteration() returns immediately. For example, when the global
+  //       planner is idling while waiting for the next waypoint to be reached.
+  ros::Rate max_rate(100 /*Hz*/);
```

Comment:
This could also be a param of GLocal system?

Reply:
I thought about making this a param but it might cause more confusion to users than it's worth, since it almost never has an effect. Perhaps a name like `max_planner_update_frequency` would be self-explanatory enough though.

Reply:
Done.